pkgs/development/tools/build-managers/scons/3.1.2.nix

```diff
@@ -36,3 +45,4 @@ python3.pkgs.buildPythonApplication {
     maintainers = with lib.maintainers; [ AndersonTorres ];
   };
 }
+# TODO: patch to get rid of distutils and other deprecations
```

Comment:
we should report that to upstream. There are thousands of packages having this problem.

Reply:
I don't believe they will care about an older release from December 2019.
It should be easier to get rid of the softwares that require Scons 3.1.2 instead.